windows-installer/setup.nsi

```diff
@@ -212,15 +217,24 @@ Section "Install"
 
     ; Trim out newlines as WordFind cannot handle them.
     ${StrTrimNewLines} $3 "$2"
-    ; Count the number of 'POS_FILE_PLACE_HOLDER' instances.  It should only
-    ; ever be 0 or 1.  We only have it in the template file once.
-    ${WordFind} "$3" "POS_FILE_PLACE_HOLDER" "#" $4
-
-    ; If we hit the place holder line replace it with the proper pos_file.
+	
+    ; Look for 'WIN_EVT_POS_FILE_PLACE_HOLDER', if found replace it with the
+    ; proper pos_file.
+    ${WordFind} "$3" "WIN_EVT_POS_FILE_PLACE_HOLDER" "#" $4
     ${If} $4 == "1"
       StrCpy $2 "  pos_file '${MAIN_INSTDIR}\pos\winevtlog.pos'$\r$\n"
     ${EndIf}
 
+    ; Look for 'CUSTOM_CONFIG_DIR_PLACE_HOLDER', if found replace it with the
+    ; proper path.
+    ${WordFind} "$3" "CUSTOM_CONFIG_DIR_PLACE_HOLDER" "#" $4
+    ${If} $4 == "1"
+	  ; Fluentd doesn't properly glob windows paths.  This is a temporary
```

Comment:
Most Windows tools I know work just fine with forward slashes instead of backslashes. Would it make sense to just always replace backslashes with forward slashes instead of relying on fluentd to fix this?